staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation.go

```diff
@@ -81,7 +81,8 @@ func validator(s *schema.Structural, isResourceRoot bool, declType *model.DeclTy
 	}
 	if len(s.Properties) > 0 {
 		propertiesValidators = make(map[string]Validator, len(s.Properties))
-		for k, prop := range s.Properties {
+		for k, p := range s.Properties {
+			prop := &p
```

Comment:
this still isn't quite right... taking the address of the iterator var is the problem (https://go.dev/play/p/wOWJw0tr_LI)

either of these would work:

```go
for k, p := range s.Properties {
  prop := p
  ...
  ... &prop
```

or

```go
for k := range s.Properties {
  prop := s.Properties[k]
  ...
  ... &prop
```

Reply:
oh yes. Thanks for catching. Fixed.